iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -52,8 +58,15 @@ class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewD
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let devotion = devotions[indexPath.row]
-        let cell = tableView.dequeueReusableCell(withIdentifier: "DevotionTableViewCell") ?? UITableViewCell()
-        cell.textLabel?.text = devotion.text
+        //swiftlint:disable force_cast
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellReusableId) as! DevotionTableViewCell
+        //swiftlint:enable force_cast
+        if let date = Formatters.devotionDateFormatter.date(from: devotion.devotion.date),
+            let day = Formatters.calendar?.components(.day, from: date).day,
+            let monthInt = Formatters.calendar?.component(.month, from: date) {
+            cell.configure(day: String(describing: day), month: Formatters.getMonth(fromInt: monthInt), verse: devotion.devotion.verse, read: devotion.read)
+
```

Comment:
whitespace. Could be cool to just say cell.configure(devotion) and let it handle everything. If it encounters any problems with dates, it can use empty fields. As it stands now, if a date error were to occur, the cell would have old values still in it from its previous position in the table view

Reply:
i've seen some quotes before in a book called Clean Code that a function should accept at most two parameters (kind of ridiculous, but something to strive for nonetheless 🐘 )